scripts/dovecot/build-obs.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Install dependencies
+echo "Installing dependencies for this script:"
+sudo apt install devscripts build-essential osc curl git debhelper-compat
+
+# Define path of your local OBS repository
+OBS_PATH=$SCRIPT_DIR/obs
+REPO_PATH=$OBS_PATH/home:deltachat/dovecot/
+SCRIPT_DIR=$PWD
```

Comment:
`SCRIPT_DIR` is used two lines above, and should be moved there, no?

Reply:
yes, although I'm surprised that it worked anyways